### ISSUE TITLE
The src path in the conf.py is not correct.

### ISSUE THREAD

**Issue by roipoussiere** [nikolavp/approval#24]
In the conf.py, you have (l. 23)

    javasphinx_command += ['-o', 'javadoc', '../../src/main/java/']

However, in `../..` there are these folders : `approval-core`, `approval-json` and `approval-sesame`, not `src`.

So the rst files can not be generated from java files with `sphinx-build`.

**Comment by nikolavp** [comment]
Yeah, I stopped generating newer versions of the documentation because javasphinx doesn't really support the newer versions of sphinx :(
